.github/workflows/ci.yaml

```diff
@@ -41,6 +41,19 @@ jobs:
 
     - run: make test generate
 
+  linux-crossversion:
```

Comment:
No strong preference, but we could also just run this on master (and even with a large branching factor), rather than on PRs too. I don't think it matters right now, as our critical path seems to be the linux-race build, which is around 8-10 mins.